meetups/2016-october/readme.md

```diff
@@ -0,0 +1,25 @@
+#Ladies of Code LDN | [October 2016](https://www.meetup.com/Ladies-of-Code-UK/events/234226231/)
+
+We had an evening of talks and discussion to demystify Hackathons - what to expect and how to get the most out of the experience.
+
+We had three presentations:
+
+###My experience of organising a hackathon - presented by [_Jessica Wade_](https://twitter.com/jesswade)
+
+Jessica shared her experience organising the "Engineer our Future" hackathon and gave us some top tips on how to turn a hackathon into a success.
+
+_Jessica has finished her PhD in the Department of Physics and Centre for Plastic Electronics at Imperial College London._
+
+###Learnings from participating in two hackathons - presented by [_Rainey David_](https://twitter.com/rainede)
+
+Rainey shared her recent experience at two hackathons by @StartupLondon and @RedMonk in September 2016. She attended both events for free as part of their diversity program to encourage more women at their events. She built [http://bit.ly/ladyproblems](http://bit.ly/ladyproblems) at a Startup Weekend event and will talk about the team's journey.
+
+The other event was a ThingMonk conference for developers with an embedded hackday where she built a voice activated light colour changing connected device.
+
+If you measure success by prizes and gifts, then in September she received an amazon echo, raspberry pi, amazon vouchers, IoT books and more. However the real prize was the experience of being there, the connections she made and the support she received from her team, the event volunteers, the mentors, the coaches and the attendees!
+
+###How to Accidentally Start a Business in 54 Hours - presented by [_Claire Mitchell_](https://twitter.com/nofootnotes) and [_Michelle Garrett_](https://twitter.com/msmichellegar)
+
+Claire and Michelle won [Startup Weekend](https://startupweekend.org/) last year, launching them on an exciting and tumultuous journey that took them to Poland and back. They’ll be sharing their experiences turning a weekend project into a real business, and their tips for hackathon success.
```

Comment:
Oops! Good spot, thanks @inespons
I'll update it tomorrow morning, no need for you to do it. Should be quick, I think.